src/main/java/com/cinema/shop/model/Ticket.java

```diff
@@ -0,0 +1,58 @@
+package com.cinema.shop.model;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
```

Comment:
Many tickets can be connected to one session. Probably, it's better to use ManyToOne annotation

Reply:
> Many tickets can be connected to one session. Probably, it's better to use ManyToOne annotation

\+